apps/app/src/labels/catalog.ts

```
@@ -1434,28 +1444,31 @@ export const appLabels = {
+  elementsAttendus: 'Éléments attendus',
+  critereAtteint: 'Critère atteint',
+  critereNonAtteint: 'Critère non atteint',
   completudeCritere:
     'Renseigner les statuts de toutes les mesures du référentiel',
-  completudeReponse: 'Ne plus avoir de statuts non renseignés',
+  completudeReponsePrefix: 'Ne plus avoir de statut',
   voirLaListe: 'Voir la liste',
   voirLaMesure: 'Voir la mesure',
   renseigner: 'Renseigner',
   chargement: 'Chargement…',
+  aCompleter: 'À compléter',
```

Biome (2.4.16)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)